Bots/DotNet/Consumers/Composer/ComposerHostBotDotNet/ComposerHostBotDotNet.botproj

```diff
@@ -0,0 +1,58 @@
+{
```

Comment:
Here are a few things to do.
- Make sure that all skills are here, i think the composer echo skill is missing, We also could add the waterfall composer skill, even if the PR isn't merged, but we ensure we don't have to do it after.
- You can use the [`Bots\DotNet\Consumers\Composer\SimpleHostBotComposer\SimpleHostBotComposer.botproj`](https://github.com/southworks/BotFramework-FunctionalTests/blob/f53af1eae65a1ecd6617c9ed41f93b8f283ea160/Bots/DotNet/Consumers/Composer/SimpleHostBotComposer/SimpleHostBotComposer.botproj) config to have similar changes, we should add the manifest and endpointName fields.